docs/core-migration-guide.md

```diff
@@ -32,6 +32,12 @@ license: |
 
 - In Spark 4.0, support for Apache Mesos as a resource manager was removed.
 
+## Upgrading from Core 3.4 to 3.5
+
+- Since Spark 3.5, `spark.yarn.executor.failuresValidityInterval` is deprecated though still works. Use `spark.executor.failuresValidityInterval` instead.
```

Comment:
`deprecated though still works.` -> `deprecated.`

Reply:
thanks for suggestion, updated. (though it was copied from the exiting one)

```
- In Spark 3.2, `spark.launcher.childConectionTimeout` is deprecated (typo) though still works. Use `spark.launcher.childConnectionTimeout` instead.
```